packages/backend/src/tests/telemetry.ts

```diff
@@ -0,0 +1,60 @@
+import { Attributes, Counter, MetricOptions } from '@opentelemetry/api'
+import { TelemetryService } from '../telemetry/service'
+import { ConvertError, Rates, RatesService } from '../rates/service'
+import { ConvertOptions } from '../rates/util'
+
+export const mockCounter = { add: jest.fn() } as Counter
+
+export class MockRatesService implements RatesService {
+  async convert(): Promise<bigint | ConvertError> {
+    return BigInt(10000)
+  }
+  async rates(): Promise<Rates> {
+    return {
+      base: 'USD',
+      rates: {
+        BGN: 0.55,
+        BNB: 249.39,
+        BTC: 40829.24,
+        ETH: 2162.15,
+        EUR: 1.08,
+        GBP: 1.25,
+        RON: 0.22,
+        USD: 1,
+        XRP: 0.5994
+      }
+    }
+  }
+}
+
+export class MockTelemetryService implements TelemetryService {
```

Comment:
will look into it, thanks!

Reply:
If you want to avoid using this class altogether (in `transaction-amount` tests), you can either just do `jest.mock` like we were chatting about earlier to define `telemetryService`, or initialize the dependencies, and just mockImplementation whenever you need to spy on a method. Up to you

Reply:
Yop, thanks. 
I decided I would keep it for now and see on further telemetry development if i should replace it with jest mocks altogheter.